common/openshift_ops.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from kubernetes import config
+from openshift.dynamic import DynamicClient
+
+log = logging.getLogger(__name__)
+
+
+class Ocp(object):
```

Comment:
Since OCP is acronym, I would have made class name all capital. 
May be we need to finalize naming conventions in coding guide as well.

Reply:
+1 on OCP instead of Ocp

Reply:
> +1 on OCP instead of Ocp

Done